pages/planetusermanual.md

```diff
@@ -32,3 +32,72 @@ Under the PLANET system, there are two types of structures: **Nations** and **Co
   * A **Community** is not able to connect to other **Communities**.
 
 The **nations** may store thousands of Resources in many formats and languages.  Unique collections of Resources can be selected and downloaded into each **Community** from the **Nation**.  This gives a Learner a personalized approach to his or her education.
+
+## How do I navigate and use PLANET?
+###_How do I access PLANET and become a member?_
+
+1. To create an account use a web browser to navigate to your Nation or Community and click on “_Become a Member_” to get started.
+
+![](uploads/images/PlanetNavigateMemberLogin.png)
+
+2. Fill out the resulting form and completing the form, click “_Register_.” You are now a member!
+
+![](uploads/images/planetnavigateAddmember.png)
+
+NOTE: Each PLANET system is unique; universal memberships do not exist.  If you become a member of a specific Nation or Community, you do not automatically become a member of any other existing Nations and Community.  To become a member of another Nation or Community you must register with a new account.
+
+###_What are the differences between Learners, Leaders, and Managers?_
+In the PLANET system, there are three different kinds of users: **Learners**, **Leaders**, and **Managers**.
+
+**Learner** is the default user level when a person first joins. Each person continues to be a Learner regardless of additional responsibilities and roles. All Learners have access to standard and individualized user functionality.
+
+**Leaders** are Learners that have created a Course or have become _Course Leaders_. Leaders have access to all Learner functionalities and to additional Course management functionalities.
+
+**Managers** have access to Learner, Leader, and additional functionalities that allow them to manage local Resources, publications, collections, surveys, data sync schedules, and local software updates.  
+If you are currently using this manual, then you are most likely a manager who is organizing and overseeing a Community or a Nation and its Communities.  In order to see the available manager functionalities, click on “Manager” next to your name on the Dashboard (see below, _How do I navigate the Dashboard_). Managers can also add, hide, delete, and view the details of Resources as well they can create, edit, re-categorize, and delete collections. 
+
+###How do I navigate the Dashboard?
+
+An example of the Manager Dashboard is shown below. The functionalities are defined for each element of the Dashboard.
+
+![](uploads/images/dashboard.png)
+
+1.	**My Home** directs the Learner to the main Dashboard page and provides easy access to most PLANET functionalities.
```

Comment:
Ok, I gave only one space between numbers and Text. Looked like this in my notepad. Should I remove all the spaces between numbers and text?

![image](https://cloud.githubusercontent.com/assets/26332034/24641159/bd2f7e9e-18c2-11e7-9d4c-c6006bc21966.png)